SPECS/python3/python3.spec

```diff
@@ -163,6 +163,8 @@ export OPT="%{extension_cflags} %{openssl_flags}"
 # and install fails. We will install these two bundled wheels manually.
 # https://github.com/pypa/pip/issues/3063
 # https://bugs.python.org/issue31916
+# Bootstrap pip3 which casues ptest build failure
```

Comment:
Please flesh out this comment a bit more-  this should mention that calling `ensurepip` like this is a no-op in the toolchain, and that this is necessary for non-toolchain builds of this package. I just don't want anyone getting confused and removing one step or another because it looks like a duplicate.